setup.py

```diff
@@ -164,9 +169,34 @@ def run(self):
 with open('README.md') as fp:
   _LONG_DESCRIPTION = fp.read()
 
+package_name = package_config.PACKAGE_NAME
+install_requires = dependencies.make_required_install_packages()
+
```

Comment:
Consider log/print the PACKAGE_NAME selected here?

Reply:
Done.